search/bwt/wavelet.go

```diff
@@ -178,7 +178,10 @@ func (wt waveletTree) Rank(char byte, i int) int {
 	}
 
 	curr := wt.root
-	ci := wt.lookupCharInfo(char)
+	ci, ok := wt.lookupCharInfo(char)
```

Comment:
In the OG BWT, we actually never used WaveletTree's select. It is semantically correct to return 0 on !ok. This can happen with a search pattern containing a character that does not exist in the original sequence.